irs-super-user.js

```diff
@@ -35,7 +35,7 @@ const registerUser = async () => {
     .initiateAuth({
       AuthFlow: 'USER_PASSWORD_AUTH',
       AuthParameters: {
-        PASSWORD: 'Testing1234$',
+        PASSWORD: 'WelcomeToFlavortown!',
```

Comment:
Should this be set to an env variable?

Reply:
This is an example script for setting up MFA for the IRS super user. This script isn't actually used in any manual or automated deploy process.